api/services/BedCountService.js

```diff
@@ -7,11 +7,11 @@ var moment = require('moment');
 const fullRange = (visibilityRange, rangeFactory) =>
   DateRange.widen.apply(null, [visibilityRange, rangeFactory(visibilityRange.from), rangeFactory(visibilityRange.to)]);
 
-let populate = (model, centreId, range) =>
+const populate = (model, centreId, range) =>
```

Comment:
Be careful with these because Rewire requires these variables to be mutable if we're trying to __set__ them